source/_docs/configuration/securing.markdown

```diff
@@ -3,14 +3,44 @@ title: "Securing"
 description: "Instructions on how to secure your Home Assistant installation."
 ---
 
-One major advantage of Home Assistant is that it's not dependent on cloud services. Even if you're only using Home Assistant on a local network, you should take steps to secure your instance.
+One major advantage of Home Assistant is that it is not dependent on cloud services.  By default, you are in control of your data. The decisions you make will affect your security and your privacy.  We take your privacy and security seriously.  Because of this, even if you are only using Home Assistant on a local network, you should take steps to secure your instance.
 
-## Checklist
+<div class='info'>
+Disclaimer: Home Assistant is not your security advisor.  Due to continually changing threat landscapes and individual configurations we recommend best practices.  The advice given here should be considered as a security baseline and should not represent your entire security profile.  
+</div>
 
-Here's the summary of what you *must* do to secure your Home Assistant system:
+### Initial Setup
+
+After installation we strongly recommend you perform the following tasks before using the system.  
+
+| Task | Justification |
+| --- | --- |
+| Setup [encrypted remote access](/docs/configuration/securing/#remote-access). | Setup HTTPS to encrypt your communication.  HTTP will allow eavesdropping on your passwords. |
+| Secure [access methods](docs/configuration/securing/#extras-for-manual-installations) | Certain installation methods have dependencies on SSH and other access methods. It is best to remove or secure these. |
+| Ensure all [Updates](https://www.home-assistant.io/common-tasks/os/#update) are applied. | Running older versions of Home Assistant can provide vulnerabilities which may be exploitable. |
+
+
+### Configuration Changes
+Configuration changes carry risk.  These risks can be reduced by performing the following tasks. 
+
+| Task | Justification|
+| --- | --- |
+| Perform pre- and post-[Snapshots](https://www.home-assistant.io/common-tasks/os/#snapshots). | In the event a configuration change needs to be reverted, you can easily revert using snapshots. |
+| Use built-in [secrets](/docs/configuration/secrets/) handling. | When requesting help from the [forums](https://community.home-assistant.io/) you will want to share your configuration. Secrets reduces your risk of exposure of sensitive data. |
+| Run [configuration checks](https://www.home-assistant.io/common-tasks/os/#configuration-check). | After making configuration changes, you should ensure your configuration is valid so it will operate as expected. | 
```

Comment:
Similar to above, this is a good practice but the configuration check won't tell you whether your instance is secured or not. If anything, putting it here may make users think that if their configuration passes checks, that their instance is secure, which is not the case

Reply:
I agree, this has no relation to security or data recovery and creates a possible false sense of security.

Reply:
It provides a low baseline to detect if the configuration will cause errors.  Errors in the configuration can prevent the system from starting properly.  Running this check can reduce risk, which is the goal of security.

Reply:
This has to be removed it has nothing to do with any of this.

Reply:
What is the purpose of configuration checks if not to ensure integrity and availability of the system?  https://www.itgovernanceusa.com/blog/how-nist-can-protect-the-cia-triad-including-the-often-overlooked-i-integrity

Reply:
Maybe I linked to the wrong page?   https://www.home-assistant.io/getting-started/configuration/

Reply:
The configuration check has nothing to do with security. Cannot add anything else. Please remove it.

Reply:
There are serious problems which can occur due to invalid configuration.    Maybe your system is more fault tolerant.  Mine is not.  I use SSH as my backup.  However, not everyone configures that by default.   



But the stock system using no integrations other than what is recommended on HA.com can have some serious problems leaving the user unable to login.  Please observe these changes which could lead to inability to use my systems. 
![image](https://user-images.githubusercontent.com/527919/108914599-96aa9400-75f9-11eb-9317-d31706093c54.png)
![image](https://user-images.githubusercontent.com/527919/108914603-9ad6b180-75f9-11eb-926d-6a9ec5301993.png)
![image](https://user-images.githubusercontent.com/527919/108914611-9d390b80-75f9-11eb-89a3-1823519933f2.png)

What is the solution if a user switches to an LDAP solution and deletes all other users, like me, then the system enters safe mode?  One might argue you should have another way to log in, and I'd agree.  But the problem could have been prevented in the first place by using Configuration Checks.

Reply:
Should this be removed, even though it can save a system from being unusable?

Reply:
Removed.